### DISCUSSION TITLE
Backstage Collaboration

### DISCUSSION THREAD

**Discussion by awanlin** [keycloak/keycloak#30900] [Q&A]
Hi, I'm an active contributor to [Backstage the framework for building an Internal Developer Portal](https://backstage.io/) which is also a CNCF project. 

We see a lot of people using Keycloak as their [auth provider](https://backstage.io/docs/auth/) for their Backstage instance. Sadly there isn't terribly good documentation on how to do this. I'm reaching out to see if the Keycloak Maintainers would like to collaborate on this with me? We'd also post about this on our blog and other applicable places once we had it in place. Perhaps as a stretch goal we could even build a dedicated Keycloak Auth Provider on the Backstage side!

This would be a great showcase of CNCF projects working together! What do you think?

**Comment by jonkoops** [comment]
Hi @awanlin, I am sure a collaboration would be invited. Try shooting an e-mail to the [development mailing list](mailto:keycloak-dev@googlegroups.com), I think you'll get a quicker response there as GitHub discussions tend to get drowned in the noise.

**Comment by stianst** [comment]
@awanlin I'm sure we can find someone that can help out - I've pinged you on Slack so we can discuss some more
